### PR TITLE
docs: Zilliz <> Arize Notebook for Search and Retrieval

### PR DESCRIPTION
**Pull request by hakantekgul**

### CODE REVIEW

File: tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,751 @@
+{
```
Comment (axiomofjoy) [inline tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb:1]: <p>Non-blocking: Might want to add the Milvus logo at the top.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (axiomofjoy) [inline tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    !pip install https://github.com/filip-halt/llama_index/archive/milvus_update.zip
</pre><p>What changes are contained on this branch from Filip? Are there plans to merge this branch into main?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (axiomofjoy) [inline tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    !pip install --upgrade --force-reinstall grpcio==1.56.0
</pre><p>Is this update of the grpcio dependency still required, or was that only needed for Milvus lite?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (axiomofjoy) [inline tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb:1]: <p>Still needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (axiomofjoy) [inline tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #45.    logging.disable(sys.maxsize)
</pre><p>What's this line doing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (axiomofjoy) [inline tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    assert openai_api_key != "", "❌ Please set your OpenAI API key"
</pre><p>This line can be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (axiomofjoy) [inline tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb:1]: <p>Non-blocking: We don't have a script for building this cloud storage asset at the moment. We will need to consolidate at some point.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (axiomofjoy) [inline tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb:1]: <p>Not clear from the instructions that the user needs to copy/paste into the cell below.</p><p> </p><p>You need a space after ## for the markdown header to be formatted properly. Same for the cells below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (axiomofjoy) [inline tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    ENDPOINT = "https://in01-a0f5cd5df959669.gcp-us-west1.vectordb.zillizcloud.com:443"
</pre><p>ENDPOINT = ""</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Review by axiomofjoy — APPROVED** [review]

**Comment by hakantekgul** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by hakantekgul** [comment]
recheck

File: tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb (hunk 1 above)
Comment (hakantekgul) [reply tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb:1]: <p>Yes, but not before the workshop</p>

Comment (hakantekgul) [reply tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb:1]: <p>this is still required</p>

Comment (hakantekgul) [reply tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb:1]: <p>Yes</p>

Comment (hakantekgul) [reply tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb:1]: <p>It is disabling any system level logs that Milvus can provide, it did before with all DEBUG workflows</p>

**Pull request merged**